src/fonts/@font-face/Demo.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>

```

Comment:
I believe you don't need this file and other files in this folder. Just add the font file itself and that's all good :)